charmcraft/parts.py

```diff
@@ -279,6 +290,7 @@ def run(self, target_step: Step) -> None:
                     self._lcm.clean(Step.BUILD, part_names=["charm"])
                     self._lcm.reload_state()
 
+            logger.debug("Executing parts lifecycle in %s", self._project_dir)
```

Comment:
We log all paths doing repr of the str, like:
```
logger.debug("Executing parts lifecycle in %r",str( self._project_dir))
```

Reply:
Fixing to keep consistency.